README.md

```diff
@@ -273,16 +273,7 @@ demonstrate that your new image can successfully build containers for each langu
 Open a PR and ping one of the Namely employees who have worked on this repo recently.  We will take
 a look as soon as we can.  Thank you!!
 
-Namely employees can merge PRs and the latest version will be pushed up via CI.  It is also possible to
-do this manually by running this:
-
-```sh
-$ make push
-```
-
-This will build and push the containers to the Namely registry located on
-[DockerHub](https://hub.docker.com/u/namely/). You must be authorized to push to
-this repo.
+Namely employees can merge PRs and cut a release/pre-release which will build & push a new image to [DockerHub](https://hub.docker.com/u/namely/) via CI.  
```

Comment:
Can you expand on `cut a release/pre-release`? Is that a branch convention or a tag?

Reply:
(I think it is a tag based on the azure pipeline, but I think it would be helpful to future devs to spell out how we create a release)